src/utils/base.test.js

```diff
@@ -0,0 +1,221 @@
+      _this.props.bgColor = _this.context.styles.colors.primary;
+      const res = fn();
+      expect(res.backgroundColor).toEqual('pink');
+    });
+  });
+
+  describe('bgGradient', () => {
+    it('should assign bgGradient value to backgroundImage', () => {
+      const bgGradientValue = 'radial-gradient(red blue green)';
+      _this.props.bgGradient = bgGradientValue;
+      const res = fn();
+      expect(res.backgroundImage).toEqual(bgGradientValue);
+    });
+  });
+
+  describe('bgImage', () => {
+    it('should assign bgImage value to backgroundImage', () => {
+      const bgImageValue = 'https://sameimage.com/url....';
+      _this.props.bgImage = bgImageValue;
+      const res = fn();
+      expect(res.backgroundImage).toEqual(`url(${bgImageValue})`);
+    });
+  });
+
+  describe('bgImage additional values', () => {
+    beforeEach(() => {
+      _this.props.bgImage = 'https://sameimage.com/url....';
+    });
+    it('should assign bgDarken value to backgroundImage opacity', () => {
+      const bgDarkenValue = 0.5;
+      const backgroundImageValue = `linear-gradient( rgba(0, 0, 0, ${bgDarkenValue}), rgba(0, 0, 0, ${bgDarkenValue}) ), url(${
+        _this.props.bgImage
+      })`;
+      _this.props.bgDarken = bgDarkenValue;
+      const res = fn();
+      expect(res.backgroundImage).toEqual(backgroundImageValue);
+    });
+
+    it('should assign bgSize value to backgroundSize', () => {
+      const bgSizeValue = 'auto';
+      _this.props.bgSize = bgSizeValue;
+      const res = fn();
+      expect(res.backgroundSize).toEqual(bgSizeValue);
+    });
+
+    it("should assign 'cover' to backgroundSize", () => {
+      const res = fn();
+      expect(res.backgroundSize).toEqual('cover');
+    });
+
+    it('should assign bgPosition value to backgroundSize', () => {
+      const bgPositionValue = 'auto';
+      _this.props.bgPosition = bgPositionValue;
+      const res = fn();
+      expect(res.backgroundPosition).toEqual(bgPositionValue);
+    });
+
+    it("should assign 'center center' to backgroundSize", () => {
+      const res = fn();
+      expect(res.backgroundPosition).toEqual('center center');
+    });
+
+    it('should assign bgRepeat value to backgroundSize', () => {
+      const bgRepeatValue = 'repeat';
+      _this.props.bgRepeat = bgRepeatValue;
+      const res = fn();
+      expect(res.backgroundRepeat).toEqual(bgRepeatValue);
+    });
+  });
+
+  describe('overflow', () => {
+    it('should assign overflow value to overflow', () => {
+      const overflowValue = 'scroll';
+      _this.props.overflow = overflowValue;
+      const res = fn();
+      expect(res.overflow).toEqual(overflowValue);
+    });
+  });
+
+  describe('height', () => {
+    it('should assign height value to height', () => {
+      const heightValue = 'scroll';
+      _this.props.height = heightValue;
+      const res = fn();
+      expect(res.height).toEqual(heightValue);
+    });
+  });
+});
+/* eslint-enable */
```

Comment:
No need to re-enable here since there's no more to this file – see comments about selective `disabling` above!